### DISCUSSION TITLE
请问海外某些自媒体对停贷的策略和建议有没有一些参考性呢？

### DISCUSSION THREAD

**Discussion by toxin08x** [WeNeedHome/SummaryOfLoanSuspension#562] [General]
虽然是反贼阵营的吧，不过她在国内有过很多年的相关经验，提出的建议会不会有一定的参考性呢？比如：

【強制停貸？】如果買到爛尾樓，如何在銀行的遊戲規則內完成技術性斷供？恆大景德鎮某樓盤全體業主想要強制停貸威脅政府和銀行，為何行不通？
https://www.youtube.com/watch?v=XHlSbqD_Ewg

连线美女博主“小翠時政財經”聊愈演愈烈的烂尾楼停贷潮：政府会怎么收拾烂摊子？业主如何跟政府博弈？
https://www.youtube.com/watch?v=a2H0LBFvZVY

爛尾樓停貸風波愈演愈烈，銀行也怕“物理攻擊”！銀行、開發商、政府，究竟誰是爛尾樓事件的首惡？
https://www.youtube.com/watch?v=Jng-5LlF928

**Comment by zxcs001** [comment]
每个人情况都不同，看看就好。

现在的情况是，很多人疫情之后顶不住了，不想断供也不得不断供了。

这一次只是顺水推舟罢了。
